### PR TITLE
Create bd-nbr.json

### PR DESCRIPTION
**Pull request by HollyODSC**
I have only one example registration number as this is a request from a new IATI Publisher.

### CODE REVIEW

File: lists/bd/bd-nbr.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      "data_not_available"
+    ],
+    "dataAccessDetails": "",
+    "features": [],
+    "licenseStatus": "no_license",
+    "licenseDetails": "No open dataset is available"
+  },
+  "meta": {
+    "source": "Original research",
+    "lastUpdated": "24-09-2024"
```
Comment (kd-ods) [inline lists/bd/bd-nbr.json:46]:
```suggestion
    "lastUpdated": "2024-09-24"
```

File: lists/bd/bd-nbr.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "company",
+    "charity",
+    "trust"
+  ],
+  "sector": [],
+  "code": "BD-NBR",
+  "confirmed": false,
```
Comment (kd-ods) [inline lists/bd/bd-nbr.json:21]:
```suggestion
  "confirmed": true,
```

We set this to true at the point of submitting the PR... and treat the PR process as the final rubber stamp.

**Review by kd-ods — CHANGES_REQUESTED** [review]
Hi @HollyODSC - just a couple of changes to make.

**Review by kd-ods — APPROVED** [review]

**Comment by kd-ods** [comment]
@HollyODSC - this is looking good. Do you want to take it out of Draft and we'll get it merged?

**Comment by HollyODSC** [comment]
> @HollyODSC - this is looking good. Do you want to take it out of Draft and we'll get it merged?

Thanks, I had put it in draft initially as we weren't sure whether to create it after some hoo-ha with the publisher, but it's good to go now!

**Pull request merged**